org.omg.sysml.jupyter.kernel/src/main/java/org/omg/sysml/jupyter/kernel/SysMLKernel.java

```diff
@@ -45,6 +45,10 @@
     private final Magics magics;
     private final MyMagicParser magicParser;
 
+    public Magics getMagics() {
```

Comment:
I don't believe this new public accessor is being used anywhere.

Reply:
@ivan-gomes, actually, I'm using these changes for other modules to test and use the original SysMLv2 kernel, and I showed these to you at Long beech meeting while we can revert these changes because the issue is solved without these changes.  But I'm not sure other problem may occur in non UTF-8 default encoding environment.

Reply:
Gotcha, in that case I will withdraw my review comment. I double checked that the `Magics` class only allows registering new magics, but not removing already registered ones, including the ones we specify, which was my initial concern with exposing it if there wasn't a use case for it.